packages/next-app/src/app/internalmanager/page.tsx

```diff
@@ -188,3 +153,100 @@ export default function Page() {
     </div>
   );
 }
+
+function TableRow({
+  token,
+  userAddress,
+  setTransaction,
+  chain,
+}: {
+  token: {
+    __typename?: "UserInternalBalance" | undefined;
+    token: `0x${string}`;
+    balance: string;
+  };
+  userAddress: `0x${string}`;
+  setTransaction: React.Dispatch<React.SetStateAction<ITransaction>>;
+  chain?: Chain;
+}) {
+  const [transferKind, setTransferKind] = useState<UserBalanceOpKind>();
```

Comment:
this sounds confusing to me since transfer is one of the operations - what about `operationKind` instead?